_component_design/selector-strip.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Selector Strip
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  Selector Strips allow the user to make a selection from a group of options. This pattern is a more visually prominent alternative to check boxes (multi-select) or radio buttons (single-select).
+preview-image: preview-images/selector-strip.svg
+status: stable
+resource: true
+last-modified: 2017-10-29
+---
+
+{% include toc.html %}
+
+## Overview
+
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+### When to use
+
+Use a selector strip when you have a small group of options the user must select from. Selector strips can be functionally identical to radio buttons or check boxes (single-select or multi-select), but are typically used instead of those patterns in situations where the selection effects the rest of the page, alters the data on the page, or where the designer would like to give more visual prominence to the group of options.
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Best practice
+
+Selector strips function best when:
+- The labels inside the strip are short (1-3 words long max) so the number of options can fit horizontally on a small screen.
```

Comment:
These should probably be bulleted with ` *`.  They're being read in as a single paragraph.

Reply:
Fixed, `-` is valid markdown syntax but there needs to be a line break after "...best when:"